docs/basics/networkbehaviour.md

```diff
@@ -109,7 +96,43 @@ If you override the virtual 'OnDestroy' method it's important to alway invoke th
 `NetworkBehaviour` handles other destroy clean up tasks and requires that you invoke the base `OnDestroy` method to operate properly.
 :::
 
-### NetworkBehaviour Pre-Spawn Synchronization
+### Pre-Spawn and MonoBehaviour Methods
+
+Since `NetworkBehaviour` is a derived `MonoBehaviour`, the `NetworkBehaviour.OnNetworkSpawn` method is treated similar to the Awake, Start, FixedUpdate, Update, and LateUpdate `MonoBehaviour` methods when the associated GameObject is active or not active in the hierarchy:
+
+- When active: Awake, Start, FixedUpdate, Update, and LateUpdate are invoked.
+- When not active: Awake, Start, FixedUpdate, Update, and LateUpdate are **not invoked**.
+
+_The unique behavior of OnNetworkSpawn, relative to the previously listed methods, is that it is not invoked until the associated GameObject is active in the hierarchy and its associated NetworkObject is spawned._
+
+If you want to disable a specific `NetworkBehaviour` but still want it to be included in the `NetworkObject` spawn process ( _i.e. so you can still enable it at a later time_), as opposed to disabling the entire `GameObject` you should disable the individual NetworkBehaviour component.
+
+:::caution
+When disabling a NetworkBehaviour component prior to spawning the `NetworkObject` you should not depend upon the `NetworkBehaviour.Start` method for any pre-initialization of properties needed by `NetworkBehaviour.OnNetworkSpawn` as `NetworkBehaviour.Start` will be invoked when the NetworkBehaviour component is enabled where `NetworkBehaviour.OnNetworkSpawn` will be invoked when spawned.
+:::
+
+:::warning Parenting, Inactive GameObjects, and NetworkBehaviour Components
+If you have child GameObjects that are not active in the hierarchy but are nested under an active GameObject with an attached NetworkObject component, then the inactive child GameObjects will not be included when spawned and for the duration of the NetworkObject's spawned lifetime. If you want all child NetworkBehaviour components to be included in the spawn process, then make sure their respective GameObjects are active in the hierarchy before spawning the `NetworkObject` or just disable the NetworkBehaviour component(s) individually.
+
```

Comment:
Here's what I understand from the info in this callout:

**Scenario**:
- You want to include all child NetworkBehaviour components in the spawning process.
- You have child GameObjects
- These child GameObjects are not active in the Hierarchy
- These child GameObjects are nested under an active GameObject
- The active GameObject (that's the parent of the child GameObjects) has a NetworkObject attached

**What happens**:
- The inactive child GameObjects won't be included when spawned for the duration of the NetworkObject component's spawned lifetime.

**Advice**:
You have two options:
1. Ensure the child GameObjects are active in the Hierarchy before spawning the NetworkObject.
2. Disable the NetworkBehaviour components individually.

Here are my questions:

- What does it mean to include a component in the spawning process?
- What would the Hierarchy in these scenarios look like?
- Is there a reason to choose one of the advice options over the other? Do they achieve the same result (include all child NetworkBehaviour components in the spawning process)? 
- What do you mean by _spawned lifetime_?

Reply:
What does it mean to include a component in the spawning process?
_This means that the NetworkObject searches for NetworkBehaviour components on its current and children GameObjects. This process excludes NetworkBehaviour components attached to disabled GameObjects._

What would the Hierarchy in these scenarios look like?
_What your actual hierarchy might look like:
- (enabled) GameObject-1 (NetworkObject)
  - (enabled) GameObject-1A (with NetworkBehaviour-1A component) 
  - (disabled) GameObject-1B (with NetworkBehaviour-1B component <---- excluded) 

How NetworkObject view the above hierarchy:
- NetworkObject
 - NetworkBehaviour-1A_

 Is there a reason to choose one of the advice options over the other? Do they achieve the same result (include all child NetworkBehaviour components in the spawning process)?
_They do achieve the same result, but under certain scenarios (i.e. in-scene placed NetworkObjects) users don't control when a NetworkObject is spawned (initially) and so that will throw everything off. Typically, if you are disabling a GameObject with NetworkBehaviours that you want to be enabled at a later time, it is easier to disable the individual NetworkBehaviour components to avoid having to learn all of the edge case scenarios._

What do you mean by spawned lifetime?
_This is basically "while the NetworkObject is spawned"...every time a NetworkObject is spawned it is assigned a unique NetworkObjectId (not to be confused with the GlobalObjectIdHash) that is used to handle messaging (i.e. send this NetworkVariable update to NetworkObjectId). When a NetworkObject is despawned, its NetworkObjectId is no longer "valid", messages no longer can be sent to it, and it is no longer considered for any form of synchronization. When despawned, if the instance is not destroyed then you can think of it like a "read to be spawned" instance...it exists...but only locally in your application instance...and it can be "respawned"...but when "respawned" it acts like it is a "new NetworkObject" (i.e. gets a new NetworkObjectId... all NetworkBehaviour components have their OnNetworkSpawn method invoked...etc.)_



Is there a reason to choose one of the advice options over the other? Do they achieve the same result (include all child NetworkBehaviour components in the spawning process)?
What do you mean by spawned lifetime?

Reply:
Thanks so much for answering all my questions, @NoelStephensUnity! I think incorporating this information into the documentation might help some users. Do you agree?

Reply:
😆 
Yes... I am going to be a bit slower than normal as I have some additional tasks added to my queue for Q2 planning.
If you are looking for a faster turn around, maybe you could add the above information in the locations called out and I could make any adjustments/additions once done? 
![image](https://user-images.githubusercontent.com/73188597/226769184-5525d466-cee6-46f8-a5e4-dbadc5944b91.png)